src/panel.js

```diff
@@ -26,6 +27,7 @@ export class LidiaPanel {
      */
     async buildSideBarPanel() {
         log("Building LIDIA panel");
+        await getAllLidiaAnnotations();
```

Comment:
This is done (and stored) in `receiveAnnotation()`, is this needed here?

Reply:
No, this was a leftover.